flocker_bb/builders/flocker_vagrant.py

```diff
@@ -379,17 +383,26 @@ def slave_class(self):
 
 ACCEPTANCE_CONFIGURATIONS = [
     AcceptanceConfiguration(
-        provider='vagrant', distribution='fedora-20'),
+        provider='vagrant', distribution='fedora-20',
+        dataset_backend='zfs'),
     AcceptanceConfiguration(
-        provider='rackspace', distribution='ubuntu-14.04'),
+        provider='rackspace', distribution='ubuntu-14.04',
+        dataset_backend='loopback'),
     AcceptanceConfiguration(
-        provider='rackspace', distribution='centos-7'),
+        provider='rackspace', distribution='centos-7',
+        dataset_backend='loopback'),
+    AcceptanceConfiguration(
+        provider='rackspace', distribution='ubuntu-14.04',
+        dataset_backend='zfs'),
+    AcceptanceConfiguration(
+        provider='rackspace', distribution='centos-7',
+        dataset_backend='zfs'),
     AcceptanceConfiguration(
         provider='rackspace', distribution='centos-7',
-        variants={'docker-head'}),
+        dataset_backend='loopback', variants={'docker-head'}),
     AcceptanceConfiguration(
         provider='rackspace', distribution='centos-7',
-        variants={'zfs-testing'}),
+        dataset_backend='zfs', variants={'zfs-testing'}),
```

Comment:
It would be useful to sort this list (in the source code) so patterns are identifiable, and exceptions obvious.  Maybe make the core (provider, distribution, backend) triple a separate loop that creates the "join" of all values.

Then comment the reasons for the remaining combinations, e.g. Vagrant box is created with Fedora and ZFS backend, why ZFS-testing is tested on CentOS but not Ubuntu?